OOP/task_4.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+class Person
+{
+    const SPECIES = "Homo Sapiens";
+
+    public function __construct(public string $name, public int $age, public string $occupation)
+    {
+    }
+
+    public function introduce()
+    {
+        return sprintf("Hello, my name is %s", $this->name);
+    }
+
+    public function describe_job()
+    {
+        return sprintf("I am currently working as a(n) %s", $this->occupation);
```

Comment:
`$this->occupation` does not exist, this code should not work
where is field declaration?